README.md

```diff
@@ -4,8 +4,12 @@ TagBot creates tags, releases, and changelogs for your Julia packages when they'
 
 When we talk about tags and releases, we mean *Git tags* and *GitHub releases*, and not releases in a registry that allow the Julia package manager to install your package.
 TagBot does not register your package for you, see the documentation in [General](https://github.com/JuliaRegistries/General/blob/master/README.md) and [Registrator](https://github.com/JuliaRegistries/Registrator.jl/blob/master/README.md) for that.
-Instead, it reacts to versions of your packages that have been registered, making TagBot a set-and-forget solution to keep your repository in sync with your package releases.
-Tags and releases aren't at all necessary, but it's considered a good practice.
+Instead, it reacts to versions of your packages that have been registered, making TagBot a set-and-forget solution to keep your repository in sync with your package releases. Tags and releases aren't at all necessary, but it's considered a good practice.
+
+### Benefits of using TagBot
+- Manual tagging and releasing on your Github repo is no longer necessary.
+- TagBot will automatically include release notes on the GitHub release page
+- Lots of configuration options (e.g. use a registry other than the General Registry)
```

Comment:
I don't usually consider the configuration options of a tool to be a benefit of that tool except when comparing it to other tools that do the same thing. It's kind of like answering "why should I use version control?" with "because Git has a lot of configuration options!".